cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -30,6 +47,74 @@ def vm(provider, small_template):
     vm.cleanup_on_provider()
 
 
+@pytest.fixture
+def cfme_vhd(provider, appliance):
+    """ Given a stream from the appliance, gets the cfme vhd"""
+    stream = appliance.version.stream()
+    try:
+        url = '{}/'.format(conf.cfme_data["basic_info"]["cfme_images_url"][stream])
+    except KeyError:
+        pytest.skip("No such stream: {} found in cfme_data.yaml".format(stream))
+    # get image name
+    image_name = get_vhd_name(url)
+    if not image_name:
+        pytest.skip("No hyperv vhd image at {}".format(url))
+    # download the image to SCVMM library
+    provider.mgmt.download_file(url + image_name, image_name)
+    provider.mgmt.update_scvmm_library()
+    yield image_name
+
+
+def create_appliance(provider, vhd):
+    """ Create an appliance from the VHD provided on SCVMM """
+    # script to create template
+    version = ".".join(re.findall(r"\d+", vhd)[0:4])
```

Comment:
Please add a comment about `[0:4]`. What is expected here?

Reply:
```python,
In [1]: import re
In [2]: vhd = "cfme-hyperv-5.10.5.0-1.x86_64.vhd"
In [3]: ".".join(re.findall(r"\d+", vhd))
Out[3]: '5.10.5.0.1.86.64'
In [4]: ".".join(re.findall(r"\d+", vhd)[0:4])
Out[4]: '5.10.5.0'
```

Reply:
so `re.findall` will give the version number, plus some extra stuff we don't care about, so I just slice the list from [0:4]. I'll add a comment about this.